.github/scripts/artifactory.spec

```diff
@@ -0,0 +1,23 @@
+{
+"files": [
+  {
+    "aql": {
+      "items.find": {
+        "repo": {"$eq":"dart.yellow.private"},
+        "path": {"$match":"*"},
+        "name": {"$match":"*"},
+        "$or": [
+          {
+            "$and": [
+              {
+                "created": { "$before":"number_days_limit" }
```

Comment:
Number_days_limit is a value where all the artifacts older than this value will be deleted.